manager/src/main/java/com/redhat/service/smartevents/manager/api/user/ProcessorsAPI.java

```diff
@@ -119,11 +120,11 @@ public Response addProcessorToBridge(@NotEmpty @PathParam("bridgeId") String bri
     @APIResponses(value = {
             @APIResponse(description = "Accepted.", responseCode = "202",
                     content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = ProcessorResponse.class))),
-            @APIResponse(description = "Bad request.", responseCode = "400", content = @Content(mediaType = MediaType.APPLICATION_JSON)),
+            @APIResponse(description = "Bad request.", responseCode = "400", content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = ErrorResponse.class))),
```

Comment:
My fear is that we return *either* `ErrorResponse` or `ErrorsResponse`.

We need support for multiple errors and hence, if limited to one response type, we should move all of RHOSE to return `ErrorsResponse` for consistency; but that puts a burden on Users and most certainly our UI development.

The same can be said for `BridgesAPI`; its methods can return multiple errors (i.e `ErrorsResponse`).

Reply:
@wtrocki @danielezonca @robpblake @kelvah FYI.

Reply:
@manstis hello, is there please a clear answer, which API methods return `ErrorResponse` and hich API methods return `ErrorsResponse`. I am asking to know, if I should:
- use both `content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = ErrorResponse.class))` and `content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = ErrorsResponse.class))`
- remove `content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = ErrorResponse.class))` completely as we do not know what object is returned at compile time

@wtrocki do you mean disabling the rule on our site? in `.github/workflows/openapi_validate.yaml`?

Reply:
yes

Reply:
tl; dr; All RHOSE API methods should return `ErrorsResponse`; affecting the whole backend and UX codebase.

@jomarko ALL of our API methods should be reviewed.

The `POST` and `PUT` methods (`createXXX` type calls) definately have the potential to return `ErrorsResponse` as a request to create something can contain multiple (validation) errors. Their current implementation return EITHER `ErrorResponse` or `ErrorsResponse`. They should probably ALWAYS return `ErrorsResponse` with a single item if there is only one error.

The `GET` methods also have the potential to return multiple Errors (since we use Java Bean Validation to validate the page number and page size). A `GET` with values `-1` for these parameters will return two validation errors.

@wtrocki @danielezonca @robpblake This blocks merge of this PR really.

Reply:
> This blocks merge of this PR really.

+1 to change everywhere to ErrorsList if you need it.

Reply:
@manstis I tried to change all API methods to return `ErrorsResponse` instead of `ErrorResponse`. My understadning is that changes are needed at two places:
- Annotations on 'API Interface'
- In real backend implementation

I tend to think the backend is already fine in:
- `ConstraintViolationExceptionMapper`

However I am not sure about my changes in:
- `ExternalUserExceptionMapper`
- `InternalPlatformExceptionMapper`

Would you mind to review my latest commit?